challenges/deploy-loderunner/README.md

```diff
@@ -0,0 +1,56 @@
+# Deploy Loderunner
+
+## Background
+
+To create a load test for our NGSA application, we need to run our end-to-end test tool (Loderunner) which generates https requests and sends them to our NGSA application. More about Loderunner can be found [here](#loderunner). 
+
+The Loderunner application needs to be deployed on our AKS cluster. Create a Kubernetes manifest file (yaml) to define and create the appropriate resource(s) needed to deploy Loderunner. The inputs needed to deploy Loderunner on our AKS cluster are shown below. Using these inputs, Loderunner will generate approximately 1 request per second when deployed. 
+
+```yaml
+
```

Comment:
based on feedback, we may change the image to ACR. the ghcr challenge will get the image into ACR

Reply:
- we removed any specific container registry reference from the challenge description and background sections 
- we are planning to record the Coaches' video using ACR instead. 
- we did not update Hints section,  command samples still referencing to "ghcr.io/retaildevcrews/webvalidate:beta"

Feedback is appreciated.